src/mam/mod_mam.erl

```diff
@@ -196,12 +197,13 @@ archive_id(Server, User)
 
 -spec start(Host :: jid:server(), Opts :: list()) -> any().
 start(Host, Opts) ->
-    ?DEBUG("mod_mam starting", []),
-    ?WARNING_MSG_IF(
-       gen_mod:get_opt(archive_groupchats, Opts, undefined) == undefined,
-       "mod_mam is enabled without explicit archive_groupchats option value."
+    ?LOG_DEBUG(#{what => mam_starting}),
```

Comment:
Maybe this could actually be an info? MAM is usually a super important functionality of a chat 🤔